pacman/pacman-canvas.js

```diff
@@ -415,7 +413,6 @@ function geronimo() {
 				var identify = new amplitude.Identify().add('games played', 1);
 				amplitude.getInstance().identify(identify);
 				addHighscore();
-				// $('#playerEmail').focus();]\
```

Comment:
why commenting out instead of removing?

Reply:
uh i just left it the way it was

Reply:
oh - i dont know - probably doesn't matter and is better to remove